### ISSUE TITLE
Elevated circulating o-Phosphoserine concentration

### ISSUE THREAD

**Issue by JillMiller** [obophenotype/human-phenotype-ontology#6045]
**Preferred term label:** Elevated circulating o-Phosphoserine concentration

**Synonyms**

**Definition (free text, please give PubMed ID)** Concentration of o-Phosphoserine in the blood is above the normal range., PMID: 3924662

**Parent term (use hpo.jax.org/app)** Abnormal circulating serine family amino acid concentration HP:0010894

**Diseases characterized by this term ? (e.g. Orphanet or OMIM number)**

**Your nano-attribution (ORCID)**

**Comment by pnrobinson** [comment]
added
